v3-nginx/Dockerfile

```diff
@@ -56,14 +57,11 @@ RUN set -eux; \
     strip /usr/local/lib/libfuzzy*.so*
 
 RUN set -eux; \
-    git clone https://github.com/SpiderLabs/ModSecurity --branch v"${MODSEC_VERSION}" --depth 1; \
+    git clone https://github.com/SpiderLabs/ModSecurity --branch v"${MODSEC_VERSION}" --depth 1 --recursive; \
     cd ModSecurity; \
+    curl -sSL https://github.com/SpiderLabs/ModSecurity/pull/2813.patch -o - | patch -p1; \
     ./build.sh; \
-    git submodule init; \
-    git submodule update; \
-    # patch make file to support Debian package names (see https://github.com/SpiderLabs/ModSecurity/issues/2810) \
-    sed -i -e 's/^\(PCRE2_POSSIBLE_LIB_NAMES=.*\)"/\1 libpcre2-8"/' build/pcre2.m4; \
-    ./configure --with-yajl=/sources/yajl/build/yajl-${YAJL_VERSION}/ --with-geoip --with-pcre2; \
+    ./configure --with-yajl=/sources/yajl/build/yajl-${YAJL_VERSION}/ --with-geoip --with-pcre2  --enable-silent-rules; \
```

Comment:
What is `enable-silent-rules`? I can't find any documentation on that.

Reply:
It's magical. Basically does `make V=0` instead of `make V=1`. So you move from
```
❯ make V=1
Making all in others
depbase=`echo libinjection/src/libinjection_html5.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
	/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../src     -g -O2 -MT libinjection/src/libinjection_html5.lo -MD -MP -MF $depbase.Tpo -c -o libinjection/src/libinjection_html5.lo libinjection/src/libinjection_html5.c &&\
	mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../src -g -O2 -MT libinjection/src/libinjection_html5.lo -MD -MP -MF libinjection/src/.deps/libinjection_html5.Tpo -c libinjection/src/libinjection_html5.c  -fno-common -DPIC -o libinjection/src/.libs/libinjection_html5.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../src -g -O2 -MT libinjection/src/libinjection_html5.lo -MD -MP -MF libinjection/src/.deps/libinjection_html5.Tpo -c libinjection/src/libinjection_html5.c -o libinjection/src/libinjection_html5.o >/dev/null 2>&1
depbase=`echo libinjection/src/libinjection_sqli.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
	/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../src     -g -O2 -MT libinjection/src/libinjection_sqli.lo -MD -MP -MF $depbase.Tpo -c -o libinjection/src/libinjection_sqli.lo libinjection/src/libinjection_sqli.c &&\
	mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../src -g -O2 -MT libinjection/src/libinjection_sqli.lo -MD -MP -MF libinjection/src/.deps/libinjection_sqli.Tpo -c libinjection/src/libinjection_sqli.c  -fno-common -DPIC -o libinjection/src/.libs/libinjection_sqli.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../src -g -O2 -MT libinjection/src/libinjection_sqli.lo -MD -MP -MF libinjection/src/.deps/libinjection_sqli.Tpo -c libinjection/src/libinjection_sqli.c -o libinjection/src/libinjection_sqli.o >/dev/null 2>&1
depbase=`echo libinjection/src/libinjection_xss.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
	/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../src     -g -O2 -MT libinjection/src/libinjection_xss.lo -MD -MP -MF $depbase.Tpo -c -o libinjection/src/libinjection_xss.lo libinjection/src/libinjection_xss.c &&\
	mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../src -g -O2 -MT libinjection/src/libinjection_xss.lo -MD -MP -MF libinjection/src/.deps/libinjection_xss.Tpo -c libinjection/src/libinjection_xss.c  -fno-common -DPIC -o libinjection/src/.libs/libinjection_xss.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../src -g -O2 -MT libinjection/src/libinjection_xss.lo -MD -MP -MF libinjection/src/.deps/libinjection_xss.Tpo -c libinjection/src/libinjection_xss.c -o libinjection/src/libinjection_xss.o >/dev/null 2>&1
/bin/sh ../libtool  --tag=CC   --mode=link gcc  -g -O2   -o libinjection.la  libinjection/src/libinjection_html5.lo libinjection/src/libinjection_sqli.lo libinjection/src/libinjection_xss.lo
libtool: link: ar cr .libs/libinjection.a libinjection/src/.libs/libinjection_html5.o libinjection/src/.libs/libinjection_sqli.o libinjection/src/.libs/libinjection_xss.o
libtool: link: ranlib .libs/libinjection.a
libtool: link: ( cd ".libs" && rm -f "libinjection.la" && ln -s "../libinjection.la" "libinjection.la" )
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../src    -D MBEDTLS_CONFIG_FILE=\"mbed-tls-config.h\" -Iothers -g -O2 -MT mbedtls/libmbedtls_la-base64.lo -MD -MP -MF mbedtls/.deps/libmbedtls_la-base64.Tpo -c -o mbedtls/libmbedtls_la-base64.lo `test -f 'mbedtls/base64.c' || echo './'`mbedtls/base64.c
```
to a
```
#14 18.90 + make install
#14 18.91 Making install in others
#14 18.91 make[1]: Entering directory '/sources/ModSecurity/others'
#14 18.92   CC       libinjection/src/libinjection_html5.lo
#14 19.25   CC       libinjection/src/libinjection_sqli.lo
#14 20.67   CC       libinjection/src/libinjection_xss.lo
#14 20.91   CCLD     libinjection.la
#14 20.99   CC       mbedtls/libmbedtls_la-base64.lo
#14 21.15   CC       mbedtls/libmbedtls_la-md5.lo
#14 21.40   CC       mbedtls/libmbedtls_la-sha1.lo
#14 21.79   CCLD     libmbedtls.la
#14 21.88 make[2]: Entering directory '/sources/ModSecurity/others'
#14 21.88 make[2]: Nothing to be done for 'install-exec-am'.
#14 21.88 make[2]: Nothing to be done for 'install-data-am'.
#14 21.88 make[2]: Leaving directory '/sources/ModSecurity/others'
#14 21.88 make[1]: Leaving directory '/sources/ModSecurity/others'
#14 21.88 Making install in src
#14 21.89 make[1]: Entering directory '/sources/ModSecurity/src'
#14 21.90 make[2]: Entering directory '/sources/ModSecurity/src'
#14 21.91   CXX      parser/libmodsecurity_la-seclang-parser.lo
#14 56.07   CXX      parser/libmodsecurity_la-seclang-scanner.lo
#14 91.66   CXX      parser/libmodsecurity_la-driver.lo
```
Which is way more succinct output, and better to read.

Reply:
Oh! It's a flag for _make_! I was thinking of ModSecurity _rules_ 🤣